Jenkinsfile

```diff
@@ -0,0 +1,45 @@
+#!groovy
+
+node('build-slave') {
+
+   currentBuild.result = "SUCCESS"
+   cleanWs()
+   try {
+
+      stage('Checkout'){
+
+         checkout scm
+       }
+
+      stage('Build'){
+
+        env.NODE_ENV = "build"
+        print "Environment will be : ${env.NODE_ENV}"
+        sh('git submodule foreach git pull origin release-1.7')
```

Comment:
Instead of hardcoding branch get from Jenkins configuration.

Reply:
This will do one time for each repo , after discussion with Maonj.